src/Menu/Filters/GateFilter.php

```diff
@@ -25,6 +25,10 @@ public function transform($item, Builder $builder)
 
     protected function isVisible($item)
     {
+        if (isset($item['model'])) {
```

Comment:
It is better to start this method with:

```
if (!isset($item['can'])) {
    return true
}
```

This way, we do not need the `! isset($item['can']) || ` part in the return statements.

Reply:
Sounds good to me. :D